src/components/Amalgamation/BusinessStatus.vue

```diff
@@ -29,99 +32,70 @@ import { AmlStatuses } from '@/enums'
-      case AmlStatuses.ERROR_LIMITED_RESTORATION:
-      case AmlStatuses.ERROR_NOT_AFFILIATED:
-      case AmlStatuses.ERROR_NOT_IN_GOOD_STANDING:
-      case AmlStatuses.ERROR_ULC_MISMATCH:
-        return 'warning'
-
-      default:
-        return 'error' // should never happen
-    }
-  }
-
-  get text (): string {
-    switch (this.status) {
-      case AmlStatuses.OK:
-        return 'Ready'
-
-      case AmlStatuses.ERROR_CCC_MISMATCH:
-      case AmlStatuses.ERROR_FOREIGN:
-      case AmlStatuses.ERROR_FOREIGN_UNLIMITED:
-      case AmlStatuses.ERROR_FUTURE_EFFECTIVE_FILING:
-      case AmlStatuses.ERROR_LIMITED_RESTORATION:
-      case AmlStatuses.ERROR_NOT_AFFILIATED:
-      case AmlStatuses.ERROR_NOT_IN_GOOD_STANDING:
-      case AmlStatuses.ERROR_ULC_MISMATCH:
-        return 'Attention Required'
+  readonly AmlStatuses = AmlStatuses
 
-      default:
-        return '(Unknown)' // should never happen
-    }
-  }
+  @Prop({ required: true }) readonly status!: AmlStatuses
 
+  /**
+   * The tooltip text for the current status.
+   * Note that the status is evaluated in amalgamation-mixin.ts.
+   */
   get tooltip (): string {
+    /* eslint-disable indent */
     switch (this.status) {
       case AmlStatuses.OK:
-        return ''
+        return 'The currently selected BC Registries account has access to this business.'
+
       case AmlStatuses.ERROR_CCC_MISMATCH:
         return 'A BC Community Contribution Company must amalgamate to form a new BC Community ' +
           'Contribution Company.'
+
       case AmlStatuses.ERROR_FOREIGN:
         return 'A foreign corporation cannot be amalgamated except by Registries staff.'
+
+      case AmlStatuses.ERROR_FOREIGN_HORIZONTAL:
+        return 'A foreign company (including an Extraprovincial Company) cannot be part of a Short ' +
+          'Form Horizontal amalgamation. '
+
       case AmlStatuses.ERROR_FOREIGN_UNLIMITED:
         return 'A foreign corporation must not amalgamate with a limited company and continue as ' +
           'an Unlimited Liability Company.'
+
+      case AmlStatuses.ERROR_FOREIGN_UNLIMITED2:
+        return 'A BC Company cannot amalgamate with an existing foreign corporation to form a BC ' +
+          'Unlimited Liability Company.'
+
+      case AmlStatuses.ERROR_FOREIGN_UNLIMITED3:
+        return 'A BC Company cannot amalgamate with a foreign company to form a BC Unlimited ' +
+          'Liability Company.'
+
       case AmlStatuses.ERROR_FUTURE_EFFECTIVE_FILING:
         return 'This business has a future effective filing. It cannot be part of an amalgamation ' +
           'until all future effective filings have come into effect.'
+
       case AmlStatuses.ERROR_LIMITED_RESTORATION:
         return 'This business is under limited restoration. It cannot be part of an amalgamation ' +
           'unless it is fully restored.'
+
+      // case AmlStatuses.ERROR_NEED_BC_COMPANY:
+      //   return 'You must add at least one BC company.'
```

Comment:
I'm still waiting to hear whether this is required. I expect the validation rules to still evolve a bit so I'm leaving this for now.